docker/Dockerfile

```diff
@@ -30,7 +26,23 @@ VOLUME /usr/local/fledge/data
 VOLUME /usr/local/fledge
 VOLUME /fledge-core/fledge
 
-EXPOSE 8081 1995
+EXPOSE 8081 1995 80
 
-CMD ["bash", "/fledge-core/start.sh"]
 
+RUN wget -q -O - http://archives.fledge-iot.org/KEY.gpg | sudo apt-key add -
+RUN echo "deb http://archives.fledge-iot.org/nightly/ubuntu1804/x86_64/ / " > /etc/apt/sources.list.d/fledge.list
```

Comment:
Do we have WL handling for this?

Reply:
Yes, it is handled in white-labeling scripts.